app.py

```diff
@@ -0,0 +1,84 @@
+import datetime
+import logging
+import os
+import typing
+
+import boto3
+from django.db import transaction  # type: ignore
+from openstates.cli.reports import generate_session_report
+
+
+logger = logging.getLogger("openstates")
+s3_client = boto3.client('s3')
+
+
+def process_upload_function(event, context):
+    """
+    Process a file upload.
+    """
+
+    # Get the uploaded file's information
+    bucket = event['Records'][0]['s3']['bucket']['name']  # Will be `my-bucket`
+    key = event['Records'][0]['s3']['object']['key']  # Will be the file path of whatever file was uploaded.
+
+    # Get the bytes from S3
+    datadir = "/tmp/"
+
+    state_acronym = key.split("/")[0]  # e.g az, al, etc
```

Comment:
I think it would be better if you called this something like jurisdiction_acronym instead of state_acronym. Its a small difference but for consistency sake with how we do other OS stuff we use jurisdiction because we get data from some jurisdictions that are not states like Puerto Rico, Federal Gov, District of Columbia

Reply:
Makes sense, done